moduleroot/.github/PULL_REQUEST_TEMPLATE.md.erb

```diff
@@ -0,0 +1,12 @@
+Please ensure that you have completed all of the following steps:
+
+- [ ] Provided your PR with a meaningful title as this will become an entry in the `CHANGELOG.md`
+- [ ] Documented your change in the `README.md` if necessary
+- [ ] Included tests for your functionality
+- [ ] Added a label to categorize your change
+- [ ] Confirmed that all tests are passing
+
+This is an example of a great [PR](https://git.io/fpRgw). For additional information please see our [contributing guidelines](https://puppet.com/docs/puppet/latest/contributing.html).
```

Comment:
Would prefer a full URL instead of URL shortening service for https://git.io/fpRgw.  Who owns that URL? who can change it? Using simple text would make it easier for source control and administration